### PR TITLE
Remove unused variable

### LINKED ISSUE #171: Mitigating security vulnerability analysis from LGTM
_Originally related to PR #170 proposed by @tdrozdovsky_

### PR DESCRIPTION
**Pull request by tdrozdovsky**
Signed-off-by: Taras Drozdovskyi <t.drozdovsky@samsung.com>

# Description

Removing an unused variable (more readable code, fix LGTM alert)

Fixes #171 

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)

# How Has This Been Tested?
[Build and run according to the documentation](https://github.com/lf-edge/edge-home-orchestration-go/blob/master/doc/platforms/x86_64_linux/x86_64_linux.md#how-to-work)

**Test Configuration**:
* Firmware version: Ubuntu 16.04
* Hardware: x86-64
* Toolchain: Docker and Go recommended versions
* Edge Orchestration Release: Coconut

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: src/controller/discoverymgr/mnedc/client/client.go [hunk 1]
```diff
@@ -140,7 +137,7 @@ func (c *Client) CreateClient(deviceID, configPath string, isSecure bool) (*Clie
 
 		log.Println("Write Successful")
 		//conn.SetReadDeadline(time.Now().Add(5 * time.Second))
-		n, readBuff, err := networkUtilIns.ReadFrom(conn)
+		readBufSize, readBuf, err := networkUtilIns.ReadFrom(conn)
```
Comment (MoonkiHong) [inline src/controller/discoverymgr/mnedc/client/client.go:140]: Good point.

**Review by MoonkiHong — APPROVED** [review]
LGTM.

**Review by t25kim — APPROVED** [review]
LGTM

File: src/controller/discoverymgr/mnedc/client/client.go (hunk 1 above)
Comment (mgjeong) [reply src/controller/discoverymgr/mnedc/client/client.go:140]: 👍

**Review by mgjeong — APPROVED** [review]

**Pull request merged**